pandas/tests/frame/methods/test_compare.py

```diff
@@ -0,0 +1,165 @@
+import numpy as np
+import pytest
+
+import pandas as pd
+import pandas._testing as tm
+
+
+@pytest.mark.parametrize("align_axis", [0, 1, "index", "columns"])
+def test_compare_axis(align_axis):
+    df = pd.DataFrame(
```

Comment:
can you add the issue number here as a comment

Reply:
done